packages/integration-karma/helpers/test-setup.js

```diff
@@ -7,52 +7,36 @@
 
 // Global beforeEach/afterEach/etc logic to run before and after each test
 
-var knownChildren;
+var knownBodyChildren;
 
-// After each test, clean up any DOM elements that were inserted into the document
-// <head> or <body>, plus any global <style>s the engine might think are there.
-
-function getChildren() {
-    return []
-        .concat(Array.prototype.slice.apply(document.head.children))
-        .concat(Array.prototype.slice.apply(document.body.children));
+// After each test, clean up any DOM elements that were inserted into the document <body>
+function getBodyChildren() {
+    return Array.prototype.slice.apply(document.body.children);
 }
 
 beforeEach(function () {
-    knownChildren = getChildren();
+    knownBodyChildren = getBodyChildren();
 });
 
 afterEach(function () {
-    getChildren().forEach(function (child) {
-        if (knownChildren.indexOf(child) === -1) {
+    getBodyChildren().forEach(function (child) {
+        if (knownBodyChildren.indexOf(child) === -1) {
             child.parentElement.removeChild(child);
         }
     });
-    knownChildren = undefined;
-    // Need to clear this or else the engine will think there's a <style> in the <head>
-    // that already has the style, even though we just removed it
-    window.__lwcResetGlobalStylesheets();
+    knownBodyChildren = undefined;
 });
 
 // Run some logic before all tests have run and after all tests have run to ensure that
 // no test dirtied the DOM with leftover elements
-var originalHeadChildren;
 var originalBodyChildren;
 beforeAll(function () {
-    originalHeadChildren = Array.prototype.slice.call(document.head.children);
-    originalBodyChildren = Array.prototype.slice.call(document.body.children);
+    originalBodyChildren = getBodyChildren();
 });
 
 // Throwing an Error in afterAll will cause a non-zero exit code
 afterAll(function () {
-    var headChildren = Array.prototype.slice.call(document.head.children);
-    var bodyChildren = Array.prototype.slice.call(document.body.children);
-
-    headChildren.forEach(function (child, i) {
-        if (originalHeadChildren[i] !== child) {
-            throw new Error('Unexpected element left in the <head> by a test: ' + child.outerHTML);
-        }
-    });
+    var bodyChildren = getBodyChildren();
```

Comment:
We actually _don't_ want the test to clear the `<head>` after every test, because that might hide bugs in the stylesheet count bookkeeping. Changing this code helped me find some bugs in how I was handling hydration, as well as the hot-swapping issue.